homeassistant/components/home_connect/select.py

```diff
@@ -265,9 +265,13 @@ async def async_update(self) -> None:
             program_translation_key := PROGRAMS_TRANSLATION_KEYS_MAP.get(program)
         ):
             _LOGGER.debug(
-                'The program "%s" is not part of the official Home Connect API specification',
+                'The program "%s" is not part of the official Home Connect API specification'
+                "and it will not be displayed, if you need to check this program, use `program_key` attribute",
                 program,
             )
```

Comment:
How often does this happen?

Reply:
I don't know, I've seen at least two cases, one is from @boergegrunicke

Reply:
For my coffee maker, this happens every time it rinses on power on, and power off, as well as anytime somebody starts a favorite.